versions.properties

```diff
@@ -11,16 +11,61 @@ plugin.com.github.johnrengelman.shadow=7.0.0
 
 plugin.com.github.vlsi.gradle-extensions=1.74
 
-plugin.com.github.vlsi.stage-vote-release=1.74
+plugin.com.github.vlsi.stage-vote-release=1.77
+##                            # available=1.75
+##                            # available=1.76
+##                            # available=1.77
```

Comment:
Remove old commented version

Reply:
Thanks @juherr - I didnt know that. Fixed